partner_create_by_vat/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+{
+    'name': 'Partner Create by VAT',
+    'summary': 'Create Partners by VAT from VIES Webservice',
```

Comment:
Using VIES webservice, name and address information will be fetched and added to the company information.